CONTRIBUTING.md

```diff
@@ -0,0 +1,69 @@
+#Contributing
+
+Thank you for your interest in contributing to Eventbrite's Javascript SDK!
+
+###Table of Contents
+* [Workflow](#workflow)
+  * [Setup](#setup)
+  * [Branches](#using-branches-to-submit-changes)
+  * [Keeping up to date](#keeping-your-local-repo-up-to-date)
+* [Creating issues](#creating-issues)
+* [Working on and submitting changes](#working-on-and-submitting-changes)
+  * [Steps to submit](#steps-to-submit)
+
+## Workflow
+
+### Setup
+
+1. Fork the repository
+2. Clone your new forked repository to your local computer
+3. Set the Eventbrite repository as your branches upstream branch
+`git remote add upstream https://github.com/eventbrite/eventbrite-sdk-javascript.git`
+4. Navigate to the root directory of your newly cloned repository 
+(for mac `cd /path/to/eventbrite-sdk-javascript`)
+5. `yarn install` to install local dependencies
```

Comment:
I guess we need to add "install Yarn" to part of the setup?

By the way, in markdown, you can number each one like `1.` and it'll create an ordered list for you. That way you can add/reorder without having to re-number

Reply:
nice tip on the list! i'll just make everything 1. I'll link docs to installing yarn